src/test/rustdoc/issue-102835.rs

```diff
@@ -0,0 +1,21 @@
+// compile-flags: -Znormalize-docs
```

Comment:
Can you name this file something more descriptive? Something like "normalize-ambiguous-item-not-fatal.rs"?